gstreamer-sdp/src/s_d_p_media.rs

```diff
@@ -465,7 +471,7 @@ impl SDPMedia {
         }
     }
 
-    pub fn set_media_from_caps(caps: &gst::Caps, media: SDPMedia) -> Result<(), ()> {
+    pub fn set_media_from_caps(caps: &gst::Caps, mut media: SDPMedia) -> Result<(), ()> {
```

Comment:
This should take the media by reference. We fill the existing media with information from the caps.